dockerfiles/base/scripts/base/library.sh

```diff
@@ -256,21 +256,28 @@ docker_compose() {
   fi
 }
 
+# Return srandom character alphanumeric string (upper and lower case) of length $1
+generate_random_string() {
```

Comment:
It's not anymore.  My first commit was going to use it with a 10 character code.  And then I decided to use `CHE_PORT` instead as the unique ID.  But I wanted to leave this utility method in the library for future use.